homeassistant/components/frontier_silicon/strings.json

```diff
@@ -1,6 +1,5 @@
 {
   "config": {
-    "flow_title": "{name}",
     "step": {
       "user": {
         "title": "Frontier Silicon Setup",
```

Comment:
This was also in the review, please remote this title, as it is already implied.